src/test/java/baza/trainee/domain/dto/EventDtoTest.java

```diff
@@ -0,0 +1,24 @@
+package baza.trainee.domain.dto;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import baza.trainee.domain.dto.event.EventDto;
+import org.junit.jupiter.api.Test;
+
+public class EventDtoTest {
+
+    @Test
+    void testEventDtoConstructorAndGetters() {
```

Comment:
Should we test constructors and Getters/Setters?

Reply:
added new test